src/test/java/emissary/test/core/junit5/LogbackTesterTest.java

```diff
@@ -90,13 +91,6 @@ void canDetectWrongThrowableMessage() {
     void canCheckOneEvent() {
         LogbackTester logBackTester = new LogbackTester(logger.getName());
         assertNotNull(logBackTester);
-        boolean[] throwing = new boolean[1];
-        String[] messages = new String[1];
-        Level[] levels = new Level[1];
-        levels[0] = Level.WARN;
-        messages[0] = "Not good news.";
-        throwing[0] = false;
-        logger.warn("Not good news.");
-        logBackTester.checkLogList(levels, messages, throwing);
+        logBackTester.checkLogList(Collections.emptyList());
```

Comment:
This isn't an equivalent change. It should be checking a list with one element, not an empty list. (The `logger.warn("Not good news")` line should not have been deleted.)